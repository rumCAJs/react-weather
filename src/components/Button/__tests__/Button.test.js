import React from 'react'
import { Button } from '../index'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(<Button>Facebook</Button>).toJSON()
  expect(tree).toMatchSnapshot()
})
