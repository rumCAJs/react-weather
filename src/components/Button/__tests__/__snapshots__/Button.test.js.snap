// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="Button-sc-1oktj3-0 ldzIPq"
>
  Facebook
</button>
`;
